### ISSUE TITLE
General explainer on AAA

### ISSUE THREAD

**Issue by bruce-usab** [w3c/wcag2ict#811]
From [conversation on call, 6 Nov](https://www.w3.org/2025/11/06-wcag2ict-minutes.html#5a5b) next draft warrants a new section providing an overview or introduction to AAA as distinct from A/AA.  This could probably be modeled after “SC problematic for closed functionally”.

### Rationale / Background

From WCAG ([&sect;5.2.1, Note 2](https://www.w3.org/TR/WCAG22/#cc1)), with AAA there is a serious caveat.
> It is not recommended that Level AAA conformance be required as a general policy for entire sites because it is not possible to satisfy all Level AAA success criteria for some content.

The most frequent reason for an SC to be at AAA is that there are known web technologies for which the SC is not feasible.  Since WCAG2ICT is targeted to non-web technologies, the AAA will be even more problematic.

By addressing AAA SC, there is an implication that AAA SC are _generally_ applicable to non-web technologies, but that is not correct.  It is useful to have the AAA guidance, but a new introductory section would serve to highlight the caution needed when applying WCAG success criteria to non-web ICT.

**Comment by GreggVan** [comment]
I agree-- this should be a separate section that leads with quoting that caveat.    It would be good to have a model for it as we dive into it  (and a place to put our results)

**Comment by bruce-usab** [comment]
Note to self from today's conversation:  AGWG concluded that WCAG 2x criteria (including AAA) are feasible (for web content) as that was part of implementation report and the Candidate Recommendation process.  This process included testing with, and support of, at least two real-world (i.e., commercially available, or open source) user agents (i.e., assistive technology and browsers).

Non-web content has had no such vetting, and such an endeavor is not in scope for WCAG2ICT.

> success criteria are written as testable statements that are **not technology-specific**

The SC are technology-specific to web content.  The comments on closed ICT include only _some_ examples where the SC (as written) might be infeasible for non-web content.  The majority[^1] of AAA SC are not applicable to a range of non-web content.  The majority[^1] of AAA SC are not supported by available assistive technology when applied to non-web content

[^1]: We will need a tally at some point.  I am guessing that it is a majority and it might be fewer than half of the AAA SC which are particularly problematic.
